### PR TITLE
TRUD data import process

### PR DESCRIPTION
**Pull request by bjss-colin**

### CODE REVIEW

File: src/OrganisationImporter/Models/OrgRefData.cs [hunk 1]
```diff
@@ -0,0 +1,305 @@
+﻿
+using System.Xml.Serialization;
```
Comment (ghost) [inline src/OrganisationImporter/Models/OrgRefData.cs:2]: FYI, this is generated using an XML to C# generator. We don't own the schema, nor do we have any control over it, so I intentionally haven't put a great deal of effort in making this more presentable.

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql [hunk 2]
```diff
@@ -1,12 +1,10 @@
 ﻿CREATE TABLE [ods_organisations].[OdsOrganisations]
 (
```
Comment (dssherlock) [inline database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql:2]: Not an issue just interested in why the Id has been upped to 10 char?

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/OdsOrganisations/Models/OdsOrganisation.cs [hunk 3]
```diff
@@ -28,7 +24,9 @@ public class OdsOrganisation
 
     public bool IsActive { get; set; }
 
-    public virtual ICollection<OrganisationRelationship> Related { get; set; }
+    public virtual ICollection<OrganisationRelationship> Related { get; set; } =
+        new HashSet<OrganisationRelationship>();
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.EntityFramework/OdsOrganisations/Models/OdsOrganisation.cs:29]: Why would an Organisation have multiple parents?

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql (hunk 2 above)
Comment (ghost) [reply database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql:2]: We originally thought it was 8 chars but there are records (organisations in the XML dataset) that contain up to 9 and 10 chars.

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/OdsOrganisations/Models/OdsOrganisation.cs (hunk 3 above)
Comment (ghost) [reply src/NHSD.GPIT.BuyingCatalogue.EntityFramework/OdsOrganisations/Models/OdsOrganisation.cs:29]: Good question. `Parent` is, for lack of a better term, the overarching organisations that relate to a given organisation.

In this context you could have Organisation A that `(RE5) IS LOCATED IN THE GEOGRAPHY OF` Organisation B whilst `(RE4) IS COMMISSIONED BY` Organisation C.

RE5 and RE4 are identifiers for the type of relationship it describes.

**Review by steves-bjss — APPROVED** [review]

**Pull request merged**